### PR TITLE
PIM-5012: Doc about how to change minimal translation ratio

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

File: cookbook/ui_customization/change_pim_locale.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+In this cookbook, you will learn how to manage Akeneo PIM locales.
+
+How to allow more locales
+-------------------------
+
+The global Akeneo PIM locale and user interface locales are configurable. By default, the Akeneo PIM is configured to
+only allow locales translated to more than 80%.
+
+If you want to allow more locales to be accessible, you can configure this limit in
+``src/Pim/Bundle/LocalizationBundle/Resources/config/providers.yml`` file:
```
Comment (skeleton) [inline cookbook/ui_customization/change_pim_locale.rst]: nope. This file will be in vendors directory, integrators don't change these files.
You have to explain what parameter you need to override. e.g.

```    
If you want to allow more locales, you can change the limit by overriding the parameter:
parameters:
    pim_localization.provider.ui_locale.min_percentage: 0.7
```

File: cookbook/ui_customization/change_pim_locale.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        pim_localization.provider.ui_locale.min_percentage: 0.8
+
+For example, if you want to allow locales translated to more than 70%, replace the minimal percentage ``0.8`` by
+``0.7``.
+
+How to improve translations
+---------------------------
+
+If your locale is not allowed or if you need to add missing translations, you can fill the related ``messages.yml`` or,
+better, contribute and :doc:`/contributing/translate`.
```
Comment (skeleton) [inline cookbook/ui_customization/change_pim_locale.rst]: ```and :doc:`/contributing/translate`. ``` are you sure about this syntax ?

Comment (pierallard) [inline cookbook/ui_customization/change_pim_locale.rst]: Seen a few times in other templates (e.g. https://github.com/akeneo/pim-docs/blob/master/contributing/create_connector.rst ... It generates a simple link)

File: cookbook/ui_customization/change_pim_locale.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
+How to change the PIM locale
+============================
+
+In this cookbook, you will learn how to manage Akeneo PIM locales.
+
+How to allow more locales
+-------------------------
+
+The global Akeneo PIM locale and user interface locales are configurable. By default, the Akeneo PIM is configured to
+only allow locales translated to more than 80%.
```
Comment (grena) [inline cookbook/ui_customization/change_pim_locale.rst]: to allow only*

File: cookbook/ui_customization/change_pim_locale.rst (hunk 2 above)
Comment (skeleton) [inline cookbook/ui_customization/change_pim_locale.rst]: IMHO, it's a typo. It's strange for a user to read ` :doc:'/cookbook/import_export/create-connector'`

Comment (pierallard) [inline cookbook/ui_customization/change_pim_locale.rst]: Only visible with Github parser, not with Sphinx.

**Comment by grena** [comment]
GTM

File: cookbook/ui_customization/change_pim_locale.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+allow only locales translated to more than 80%.
+
+If you want to allow more locales, you can change the limit by overriding the parameter:
+
+.. code-block:: yaml
+
+    parameters:
+        pim_localization.provider.ui_locale.min_percentage: 0.7
+
+In this example, you will allow locales translated to more than 70%.
```
Comment (phaseinducer) [inline cookbook/ui_customization/change_pim_locale.rst]: Just "you allow" ?

**Comment by phaseinducer** [comment]
GTM

File: cookbook/ui_customization/change_pim_locale.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
+How to change the PIM locale
+============================
+
+In this cookbook, you will learn how to manage Akeneo PIM locales.
+
+How to allow more locales
+-------------------------
+
+The global Akeneo PIM locale and user interface locales are configurable. By default, the Akeneo PIM is configured to
+allow only locales translated to more than 80%.
```
Comment (fred2gombert) [inline cookbook/ui_customization/change_pim_locale.rst]: You should precise what means 80% (80% in Crowdin) and maybe provide a link

**Pull request merged**